src/packages/core/components/input-with-alias/input-with-alias.element.ts

```diff
@@ -68,17 +68,21 @@ export class UmbInputWithAliasElement extends UmbFormControlMixin<string>(UmbLit
 	}
 
 	render() {
-		// Localizations: [NL]
 		return html`
-			<uui-input id="name" placeholder="Enter a name..." label=${this.label} .value=${this.value} @input="${this.#onNameChange}">
+			<uui-input
+				id="name"
+				placeholder=${this.localize.term('placeholders_entername')}
+				label=${this.label}
```

Comment:
And copy the placeholder localization here as well

Reply:
@iOvergaard Sure, but the `label` property (for the "name" input) is set on the parent element, e.g. for the Document Type Workspace: https://github.com/umbraco/Umbraco.CMS.Backoffice/blob/main/src/packages/documents/document-types/workspace/document-type-workspace-editor.element.ts#L88

Did you mean add the localization on the parent element? (_This would be for Document Type, Media Type and Member Type workspaces_).

Reply:
Updated, to add the localization on the parent component for the `label` text, with a fallback in the `umb-input-with-alias` to use the same localization (e.g. if it's `undefined` or someone wants to use different text/localization). 👍